tm2/pkg/bft/consensus/state.go

```diff
@@ -666,6 +691,11 @@ func (cs *ConsensusState) receiveRoutine(maxSteps int) {
 
 // state transitions on complete-proposal, 2/3-any, 2/3-one
 func (cs *ConsensusState) handleMsg(mi msgInfo) {
+	span := cs.addTrace(fmt.Sprintf("handleMsg(peerID:%s)", mi.PeerID))
```

Comment:
why not use attributes here (and everywhere else)? 
```suggestion
	span := cs.addTrace("handleMsg", trace.WithAttributes(attribute.String("PeerID", mi.PeerID))))
```
Using attributes could later help to better identify the trace given specific attributes.

Reply:
I've added attributes in `addTrace` (I will add some more attrs like you suggest) for tracing but here I keep some short info in the label to have more visibility when seeing the traces (like in the screenshot).
I can remove those labels and keep only function name as the span name if we think that's not necessary ?

wdyt ?

Reply:
Yes, sure! You can include brief information in the name to help identify the trace, but please keep it concise for readability. Feel free to duplicate this information as an attribute as well. If I remember correctly (this was the case in Jaeger) when you click on a trace, you should see the attributes, which is very helpful for understanding the context.